antarest/study/business/output_variables_management.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from pathlib import Path
+from typing import Any, Iterator
+
+from antarest.study.business.aggregator_management import (
+    AggregatorManager,
+    MCAllAreasQueryFile,
+    MCAllLinksQueryFile,
+    MCIndAreasQueryFile,
+    MCIndLinksQueryFile,
+    MCRoot,
+    QueryFileType,
+)
+from antarest.study.storage.output_model import OutputVariablesMetadata
+from antarest.study.storage.rawstudy.model.filesystem.matrix.matrix import MatrixFrequency
+
+
+class OutputVariablesManager:
```

Comment:
We could have only free functions here, not a class.
It's useful to have a class when you have some state to encapsulate, but it's not the case here